### PR TITLE
[FEATURE] Add T3kit10 php7.4 apache 2.4 buster docker files

### PR DESCRIPTION
**Pull request by pixelmatseriks**
# New Pull Request checklist

## Please check if your PR fulfills the following requirements

- [x] Contributing to t3kit: <https://github.com/t3kit/.github/blob/master/CONTRIBUTING.md>
- [x] Coding Rules: <https://github.com/t3kit/.github/blob/master/CONTRIBUTING.md#coding-rules>
- [x] Commit message conventions: <https://github.com/t3kit/.github/blob/master/CONTRIBUTING.md#commit-message-guidelines>

## PR Type

What kind of change does this PR introduce?

- [x] [FEATURE] - A new feature
- [ ] [BUGFIX] - A bug fix
- [ ] [TASK] - Any task, which is not a **new feature** or **bugfix**
- [ ] [TEST] - Adding missing tests
- [ ] [DOC] - Documentation only changes
- [ ] [WIP] - Work in progress tag, should not be present when creating pull requests
- [ ] [SECURITY] - To mark important updates, when evaluating when we should update older sites

## Breaking change

Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Issue references

Issue Number: #...

### CODE REVIEW

File: t3kit10/php7.4-apache2.4-buster/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,89 @@
+###########################################################
+# Web server docker image for t3kit project
+# Web server docker image for TYPO3
+#
+# t3kit/10-php7.3-apache2.4-stretch
```
Comment (dmh) [inline t3kit10/php7.4-apache2.4-buster/Dockerfile]: t3kit/10-php7.4-apache2.4-buster

File: t3kit10/php7.4-apache2.4-buster/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    # Install libs
+    libxml2-dev \
+    libfreetype6-dev\
+    libjpeg-dev \
+    libpng-dev \
+    libzip-dev \
+    # Install tools
+    nano=3.2-* \
+    graphicsmagick=1.4* \
+    imagemagick=8:6.* \
```
Comment (dmh) [inline t3kit10/php7.4-apache2.4-buster/Dockerfile]: we can skip imagemagick since Platform doesn't support this. What do you think?

**Review by dmh — CHANGES_REQUESTED** [review]
Can you also add hadolint check in code-guidelines.yml

**Comment by pixelmatseriks** [comment]
> Can you also add hadolint check in code-guidelines.yml
the hadolint check is added in code-guidelines.yml

**Review by dmh — APPROVED** [review]

**Pull request merged**